doc/user/cluster_tls.md

```diff
@@ -56,56 +54,66 @@ The client TLS assets should have the following:
+
 ```
 $ kubectl create secret generic etcd-server-tls --from-file=server-ca.crt --from-file=server.crt --from-file=server.key
 ```
 
 etcd-operator will mount this secret at `/etc/etcdtls/member/server-tls/` for each etcd member pod in the cluster.
 
-
 ### operatorSecret
 
-Operator needs to send client requests e.g. snapshot, healthy check, add/remove member in order to maintain this cluster.
+The operator must send the client requests (such as snapshot, healthy check, add/remove member) to maintain this cluster.
+
 `operatorSecret` contains pem-encoded private keys and x509 certificates for communicating with etcd server via client URL.
 
 The operator's etcd TLS assets should have the following:
 - **etcd-client.crt**: operator's etcd x509 client cert.
 - **etcd-client.key**: operator's etcd x509 client key.
 - **etcd-client-ca.crt**: CA cert for validating the certs of etcd members.
-They corresponds to the `--cert`,`--key`, and `--cacert` arguments of `etcdctl`.
+They correspond to the `--cert`,`--key`, and `--cacert` arguments of `etcdctl`.
+
+Create a secret containing the `etcd-client.crt`, `etcd-client.key`, and `etcd-client-ca.crt`:
 
-Create a secret containing those:
 ```
 $ kubectl create secret generic etcd-client-tls --from-file=etcd-client-ca.crt --from-file=etcd-client.crt --from-file=etcd-client.key
 ```
 
-Pass `etcd-client-tls` to `operatorSecret` field.
+Pass `etcd-client-tls` to the `operatorSecret` field.
 
 ### Access a secure etcd cluster
 
 Assume a secure etcd cluster `example` is up and running.
 
 To access the cluster, use the service `example-client.default.svc`, which matches the SAN of its certificates.
 
-Assume the following certs are being used:
+Assume the following certs are used:
+
 ```
 etcd-client.crt
 etcd-client.key
 etcd-client-ca.crt
 ```
-Both `etcd-client.crt` and `etcd-client.key` should trusted by etcd server's client CA `server-ca.crt`.
-`etcd-client-ca.crt` should trust etcd server's key and cert.
 
-Here is an example `etcdctl` command to list members from the secure etcd cluster:
+Both `etcd-client.crt` and `etcd-client.key` should be trusted by the etcd server's client CA `server-ca.crt`.
+
+`etcd-client-ca.crt` should trust the etcd server's key and cert.
+
+The following  example `etcdctl` command will list members from the secure etcd cluster. Run this command from within a Kubernetes Pod to access the service name:
+
 ```
 $ ETCDCTL_API=3 etcdctl --endpoints=https://example-client.default.svc:2379 \
     --cert=etcd-client.crt --key=etcd-client.key --cacert=etcd-client-ca.crt \
     member list -w table
 ```
-It should be run within a pod inside k8s in order to access the service name.
+
+
+[etcd-security]: https://coreos.com/etcd/docs/latest/op-guide/security.html
+[self-signed]: https://coreos.com/os/docs/latest/generate-self-signed-certificates.html
+[example-tls]: https://github.com/coreos/etcd-operator/tree/master/example/tls/
```

Comment:
use relative path link:
```
../../example/tls/
```

Reply:
I don't think that will work when these docs are published to the site. I think we have to use the absolute path for these. I will check.